### ISSUE TITLE
Since symfony/dependency-injection 6.4: "Symfony\Bridge\Doctrine\DataFixtures\ContainerAwareLoader" is deprecated, use dependency injection in your fixtures instead. 

### ISSUE THREAD

**Issue by ruudk** [doctrine/DoctrineFixturesBundle#420]
How can we silence this deprecation? We're not using the `ContainerAwareLoader`.

We're using:
```
symfony/doctrine-bridge               v6.4.0
doctrine/data-fixtures                1.7.0
doctrine/doctrine-fixtures-bundle     3.5.1
```

I saw this problem being predicted in https://github.com/doctrine/DoctrineFixturesBundle/pull/395 but I don't think there is a solution yet.

**Comment by derrabus** [comment]
The warning should be gone if you upgrade the bridge to v7, but I realize that this might not be the answer you're looking for. 🙈 

Maybe we could add an option to opt-out of using that class?

**Comment by BackNot** [comment]
I guess we need to wait for the 4.0 version

**Comment by ruudk** [comment]
Upgrading to 7.0 solves it for me.

**Comment by derrabus** [comment]
Thanks for the feedback, but let's leave the issue open. We should try to find a solution for Symfony 6.4 projects.

**Comment by stanislav-oop** [comment]
Is this fixed yet?

**Comment by derrabus** [comment]
No.

**Comment by Natizyskunk** [comment]
Still no solution ?
I would prefer not to mix package versions in my composer.json sinc all my project dependencies are in 6.4.

**Comment by greg0ire** [comment]
And I would prefer if you folks explored the issue and proposed solutions instead of demanding updates and filling my inbox.

**Comment by stanislav-oop** [comment]
Due to some internal company policies we can not update to anything but latest symfony LTS version which is 6.4.
Nothing of proposed solutions works for us so we would like to have this depreciation silenced. Its not a demand though.

**Comment by derrabus** [comment]
Folks.

* Nobody says that you cannot use Symfony 6.4 anymore.
* Nobody's actively working on a solution yet.
* Anyone can open a PR and propose a solution.

I hope that answers all open questions. :v:

**Comment by sofigio** [comment]
Any way to have this feature on Ubuntu 24, without waiting for the next LTS release in 2026? Like porting it to the 1.4.x... or is there any APT repository available with 1.5?
